backend/layers/common/validation.py

```diff
@@ -29,6 +29,33 @@
 }
 
 
+def _verify_field(
```

Comment:
Moved [existing](https://github.com/chanzuckerberg/single-cell-data-portal/pull/7214/files#diff-fbff6622cc471e7a74db034a50d63b58f01e8b524aee67833ade64bb8fa53a6fL42) `check` method here for reuse, updated to include `or key == "title"`.